src/commands/viewProperties.ts

```diff
@@ -3,25 +3,18 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
+import { AppSettingsTreeItem } from 'vscode-azureappservice';
 import { IActionContext, openReadOnlyJson } from 'vscode-azureextensionui';
 import { ext } from '../extensionVariables';
 import { IAzureResourceTreeItem } from '../tree/IAzureResourceTreeItem';
 import { StaticWebAppTreeItem } from '../tree/StaticWebAppTreeItem';
-import { localize } from '../utils/localize';
-import { nonNullProp } from '../utils/nonNull';
 
-export async function viewProperties(context: IActionContext, node?: IAzureResourceTreeItem): Promise<void> {
-    if (!node) {
-        node = await ext.tree.showTreeItemPicker<StaticWebAppTreeItem>(StaticWebAppTreeItem.contextValue, context);
+export async function viewProperties(context: IActionContext, target?: IAzureResourceTreeItem | AppSettingsTreeItem): Promise<void> {
```

Comment:
Lol I honestly didn't realize you added "View properties" to application settings. You went hog-wild with "View properties", didn't you? Unfortunately, the name doesn't make sense to me in this scenario. I'm viewing the settings themselves, not properties on the settings. If you want to make it more discoverable, I would maybe do "View as JSON"

Reply:
Hmm, should I just get rid of the `View as Json`?  I realized that this will end up being similar to "Download Remote Settings..." so it seems like a redundancy.

Reply:
Do we already support "Download Remote Settings..."? If yes, you could look at telemetry of both to help us decide. Otherwise yeah I'm fine removing it

Reply:
We don't support it yet, but I believe we have plans to for v0.2.0.